spec/features/callback_spec.rb

```diff
@@ -36,6 +35,7 @@
   }
 
   before do
+    ideal = Spree::PaymentMethod::SisowBilling::Ideal.create!(name: 'iDeal')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.